torch/csrc/jit/ir/graph_utils.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include <torch/csrc/jit/ir/ir.h>
+
+#include <vector>
+
+namespace torch {
+namespace jit {
+
+TypePtr getTensorType(const at::Tensor& t, bool complete);
```

Comment:
I think you need to prefix these with `TORCH_API`

Reply:
Updated in the latest version